lib/WebSocketServer.js

```diff
@@ -103,27 +103,18 @@ util.inherits(WebSocketServer, EventEmitter);
  */
 WebSocketServer.prototype.close = function (cb) {
   // terminate all associated clients
-  var error = null;
-
   if (this.clients) {
-    for (const client of this.clients) {
-      try {
-        client.terminate();
-      } catch (e) {
-        error = e;
-      }
-    }
+    for (const client of this.clients) client.terminate();
```

Comment:
I've removed the `try...catch` statement because `WebSocket.prototype.terminate()` does not throw errors.